esphome/components/gp8211/gp8211.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "esphome/components/i2c/i2c.h"
+#include "esphome/core/component.h"
+
+namespace esphome {
+namespace gp8211 {
+
+enum GP8211Voltage {
+  GP8211_VOLTAGE_5V = 0x00,
+  GP8211_VOLTAGE_10V = 0x11,
```

Comment:
You might as well make these the value of the register then you dont need the ternary in setup and can just write directly.
```suggestion
  GP8211_VOLTAGE_5V = 0x55,
  GP8211_VOLTAGE_10V = 0x77,
```